.github/workflows/ci.yml

```diff
@@ -41,3 +41,34 @@ jobs:
       with:
         flags: unittests
         fail_ci_if_error: true
+
+
+  provider-verification:
+    name: Publish Provider Verification Results
+    runs-on: ubuntu-20.04
+    needs: run_tests
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install pip
+      run: pip install -r requirements/pip.txt
+
+    - name: Install Dependencies
+      run: |
+        pip install -r requirements/ci.txt
+        pip install -r requirements/test.txt
+
+    - name: Publish Results
+      run: |
+        export PACT_BROKER_BASE_URL='https://edx.pactflow.io'
+        export PACT_BROKER_TOKEN=${{ secrets.PACT_FLOW_ACCESS_TOKEN }}
+        export PUBLISH_VERSION=`git rev-parse --short HEAD`
+        export PUBLISH_VERIFICATION_RESULTS=true
+        pytest edxval/pacts/verify_pact.py --ds=edxval.settings.pact
```

Comment:
Add `-s` flag so that logs/console statements are visible.

Reply:
Wouldn't that diable all the capturing? https://docs.pytest.org/en/6.2.x/capture.html#setting-capturing-methods-or-disabling-capturing

Reply:
Oh. On local, using -s yielded in a lot more log output, so I thought it was giving more output. I didn't consult the documents.